### PR TITLE
🚀 Replacing govuk_template ☠️

### PR DESCRIPTION
**Pull request by aliuk2012**
## Blocked by
- Phase banner & User research banner have been replaced with govuk-frontend phase banner component in all the other applications.

## What has changed?
- Removes dependency on GOV.UK Template
- Uses GOV.UK Frontend template
- Removes temporary hacks needed to ensure a smooth transition

## Other improvements
- Removes all JavaScript not needed by digitalmarketplace-user-frontend
- Removes dependency on **jQuery** 😍

## Performance impact:

|                           | Current | New   | % difference  |
|---------------------------|---------|-------|---------------|
| Requests                  | 13      | 7     | 46% reduction | 
| Transferred (Gzip)        | 467KB   | 172KB | 63% reduction |
| Transferred (Uncompressed)| 729KB   | 295KB | 60% reduction |

## Before screenshot
image

## After screenshot
image

## Lighthouse Reports (Snapshot):
🤮[Preview](https://googlechrome.github.io/lighthouse/viewer/?gist=c871de1340b97c6b60841436a26e843f)
🤩[Localhost](https://googlechrome.github.io/lighthouse/viewer/?gist=e23cae940c3dfe2f7bb4a81b0cda765f)

Ticket: https://trello.com/c/lGqdZbhS

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -4,68 +4,6 @@
...
-    }
-
-  }
-
-  GOVUK.GDM = GDM;
-
-  if (GDM.debug) console.timeEnd("Modules loaded");
-
-}).apply(this, [GOVUK||{}, GOVUK.GDM||{}]);
+GOVUKFrontend.initAll()
```
Comment (katstevens) [inline app/assets/javascripts/application.js]: 😻

**Comment by aliuk2012** [comment]
GOV.UK Frontend footers are larger by default. https://design-system.service.gov.uk/components/footer/

**Comment by lfdebrux** [comment]
I don't think we should merge this until we're ready with the other apps.

File: app/templates/_base_page.html [hunk 2]
```diff
@@ -1,38 +1,62 @@
...
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    "tag": {
+      "text": "beta"
+    },
+    "html": 'Help us improve the Digital Marketplace - <a class="govuk-link" href="'  + url_for('external.help') + '">send your feedback</a>'
+  }) }}
+  {% block breadcrumb %}{% endblock%}
+  {% include "toolkit/flash_messages.html" %}
```
Comment (lfdebrux) [inline app/templates/_base_page.html]: Flash messages should be in `content` block so it doesn't get skipped by the skip link

**Comment by lfdebrux** [comment]
It's not a super big deal but while playing around with flash messages I noticed this:

flash message with link styled with default browser style

Maybe we should add the default styles flag to this PR just in case?

**Pull request merged**